README.md

```diff
@@ -2,7 +2,7 @@
 
 macOS [![CircleCI](https://circleci.com/gh/pointfreeco/swift-overture.svg?style=svg)](https://circleci.com/gh/pointfreeco/swift-overture) Linux [![Build Status](https://travis-ci.org/pointfreeco/swift-overture.svg?branch=master)](https://travis-ci.org/pointfreeco/swift-overture)
 
-A library for function composition.
+A library for [function composition](https://www.pointfree.co/episodes/ep11-composition-without-operators).
```

Comment:
I think inline linking is a lil tricky. Would maybe rather have some direct callouts so folks don't think they're clicking a link to something like Wikipedia. I wonder if we wanna just link directly in each section. Something like:

> These functions were explored in [this episode of Point-Free](#TODO).

Feels a bit more honest and actually calls out the episodes publicly, which is nice to those that are interested that may have missed it if hidden in a link. What do you think?

Reply:
good point, will take a pass at expanding these a bit.